website/docs/installation/configuration.md

```diff
@@ -2,28 +2,54 @@
 title: Configuration
 ---
 
-These are all the configuration options you can set via environment variables.
+This page details all the authentik configuration options that you can set via environment variables.
 
-Append any of the following keys to your `.env` file, and run `docker-compose up -d` to apply them.
+import Tabs from '@theme/Tabs';
```

Comment:
to use mdx you need to rename the file to `.mdx`

Reply:
Ach. I remembered how to add a redirect, to handle the changed file name, but... I foolishly included it in this same PR and I think it needs to be a separate PR on the `netlify.toml` file and have that PR merged before this PR can pass build. I'll go now and create that PR, and try to cherrypick out the commit for that toml file in this PR... Jeesh I do keep things exciting.